src/tokens-studio/spectrum2-non-colors/spectrum2/layout.component/mobile.json

```diff
@@ -3650,8 +3860,28 @@
       }
     }
   },
-  "slider-track-thickness": {
-    "value": "2px",
+  "slider-track-height-medium": {
+    "value": "4px",
+    "type": "sizing",
+    "$extensions": {
+      "spectrum-tokens": {
+        "uuid": "",
+        "name": "slider-track-height-medium"
+      }
+    }
+  },
+  "slider-track-height-large": {
+    "value": "16px",
+    "type": "sizing",
+    "$extensions": {
+      "spectrum-tokens": {
+        "uuid": "",
+        "name": "slider-track-height-large"
+      }
+    }
+  },
+  "🚫 slider-track-thickness": {
+    "value": "{slider-track-height-medium}",
```

Comment:
same issue: https://github.com/adobe/spectrum-tokens-studio-data/pull/160/files#diff-f187071ce95a0cd7c8f1f7ea0df491cf2da079087cb666d1990fa8ccbea8011fR3883

Reply:
I have changed the value of "🚫 slider-track-thickness" back to "2px" here as well.